benches/mantis/jobs/mantis.ncl

```diff
@@ -121,47 +139,60 @@ fun params =>
 
       #same?
       # #baseTags: [namespace, #role, "mantis-${NOMAD_ALLOC_INDEX}"]
-    } & (if params.role == `passive then {
-      #TODO: dependent if
-      service."%{params.namespace}-mantis-%{string.from_enum params.role}-rpc" = {
-        address_mode = `host,
-        port = "rpc",
-        tags = [
+    }
+    & (if params.role == `passive then
+      {
+        #TODO: dependent if
+        service."%{params.namespace}-mantis-%{string.from_enum params.role}-rpc" = {
+          address_mode = `host,
+          port = "rpc",
+          tags = [
             "rpc",
             "ingress",
             "traefik.enable=true",
             "traefik.http.routers.%{params.namespace}-mantis-%{string.from_enum params.role}.rule=Host(`%{params.namespace}-%{string.from_enum params.role}.%{params.fqdn}`)",
             "traefik.http.routers.%{params.namespace}-mantis-%{string.from_enum params.role}.entrypoints=https",
             "traefik.http.routers.%{params.namespace}-mantis-%{string.from_enum params.role}.tls=true",
-        ] @ baseTags,
+          ]
+          @ baseTags,
+        }
       }
-    } else {})
+    else
+      {})
     & {
       #TODO: dependent if
       #For now, redefining namespace
       service."%{params.namespace}-mantis-%{string.from_enum params.role}-rpc" = {
         check.rpc = {
           address_mode = `host,
-            interval = "10s",
-            port = "rpc",
-            timeout = "3s",
-            type = `http,
-            path = "/healthcheck",
-          } & (if params.network != "etc" then {
-              check_restart = {
-                limit = 5,
-                grace = "10m",
-              }
-            } else {}),
+          interval = "10s",
+          port = "rpc",
+          timeout = "3s",
+          type = `http,
+          path = "/healthcheck",
+        }
+        & (if params.network != "etc" then
+          {
+            check_restart = {
+              limit = 5,
+              grace = "10m",
+            }
+          }
+        else
+          {}),
```

Comment:
Nitpick: Maybe we should (under some conditions?) make sure that `if` starts on a new line? This formatting isn't bad, but I wonder if
```
& (
  if params.network != "etc" then
    {
      check_restart = {
        limit = 5,
        grace = "10m",
      }
    }
  else
    {}),
```
might be better.

Reply:
This one stood out to me, too; but more specifically at the closing paren, which looks weird to me. Incidentally, in this case, are the parens even necessary?

Reply:
Yes, the parens are necessary for parsing. Maybe the closing paren should also go on a new line? I'm really not sure...

Reply:
Rather than targetting the conditional expression, I've added a rule that will create a new indentation block between parens when the contents cover multiple lines.

I haven't applied it here, yet, but you can see the [effect on the stdlib here](https://github.com/tweag/topiary/commit/d7c038bbcabaa5de7add34bbddce489291dfe7f8#diff-6986669f19c1caad9af03e15df1e512864cc90f72071dee180866ea9dc1210e0). In the above example, the output looks like this:

```nickel
  # etc.
  service."%{params.namespace}-mantis-%{string.from_enum params.role}-rpc" = {
    check.rpc = {
      address_mode = `host,
      interval = "10s",
      port = "rpc",
      timeout = "3s",
      type = `http,
      path = "/healthcheck",
    }
    & (
      if params.network != "etc" then
        {
          check_restart = {
            limit = 5,
            grace = "10m",
          }
        }
      else
        {}
    ),
  }
  # etc.
```

How does that look, to you? :slightly_smiling_face:

Reply:
Yeah, I think it's an improvement too :tada:

I'll run it against this repo again...